daemon/daemon.go

```diff
@@ -238,14 +228,19 @@ configRetry:
 				time.Sleep(1 * time.Second)
 				continue configRetry
 			}
-			configParams.UpdateFrom(globalConfig, config.DatastoreGlobal)
-			configParams.UpdateFrom(hostConfig, config.DatastorePerHost)
+			_, err = configParams.UpdateFrom(globalConfig, config.DatastoreGlobal)
+			if err != nil {
+				log.WithError(err).Error("Failed update global config from datastore")
```

Comment:
Now you're no longer checking `configParams.Err` below, this code needs to do the sleep and continue

Reply:
Fixed